app/helpers/tutors_helper.rb

```diff
@@ -99,9 +100,15 @@ def affiliate
     ]
   end
 
+  def language_proficiencies
+    # Returns a hash like { "English => "None", "Spanish" => "None" }
+    Hash[(native_language.map { |e| e[0] }).collect { |v| [v, 'None'] }]
```

Comment:
The tutor will have language proficiencies, and each language will have a related value (none through native). This will be the default hash. It will be modified with the input from the form and persisted to the db as a serialized hash.